subiquity/server/controllers/install.py

```diff
@@ -389,12 +389,13 @@ async def run_curtin_step(name, stages, step_config, source=None):
         rp = fs_controller.reset_partition
         if rp is not None:
             mounter = Mounter(self.app)
-            async with mounter.mounted(rp.path) as mp:
-                await run_curtin_step(
-                    name="populate recovery", stages=["extract"],
-                    step_config=self.rp_config(logs_dir, mp.p()),
-                    source='cp:///cdrom',
-                    )
+            rp_target = os.path.join(self.app.root, 'rp')
```

Comment:
how about /recovery maybe?

Reply:
Well. The terminology gets a bit confusing around this. "recovery" in some context means things like recovering your bitlocker / luks passphrase, and I've been trying to not talk about the work we are doing for OEM as a recovery partition. I think talking about it as the "factory reset" or "restore" partition is a bit less ambiguous. Fortunately (?) recovery, reset and restore all start with 'r' so talking about it as 'rp' avoids making any decisions :-)

If /rp is too obscure (which I am sympathetic with!), how about "/factory-reset"?

Reply:
Yes, I think `/factory-reset` best captures the spirit of what this partition does, I suggest going with that.

Reply:
Sorry for the delay - yes /factory-reset seems right :)